components/collection/unlockable/UnlockableLoader.vue

```diff
@@ -0,0 +1,108 @@
+          fill="#FF7AC3"
+          fill-opacity="0.99" />
+        <path
+          d="M374.508 131.524C374.508 136.527 370.46 140.583 365.468 140.583C360.476 140.583 356.429 136.527 356.429 131.524C356.429 126.521 360.476 122.465 365.468 122.465C370.46 122.465 374.508 126.521 374.508 131.524ZM243.908 21.1043C240.395 17.6797 235.539 15.4387 230.235 14.7951C230.16 14.7861 230.087 14.7856 230.013 14.7794C225.236 14.1184 222 11.0727 222.284 8.72592C222.7 5.27997 220.25 2.14801 216.812 1.73049C213.377 1.31633 210.248 3.76829 209.832 7.21367C208.644 17.0411 216.942 25.8605 228.723 27.2916C228.789 27.2994 228.853 27.2977 228.919 27.3033C231.4 27.6419 233.667 28.6572 235.163 30.1146C235.799 30.7347 236.841 31.9817 236.675 33.3613C236.508 34.7398 235.198 35.7007 234.433 36.1507C232.586 37.2354 230.062 37.6803 227.505 37.3691C222.199 36.7238 216.951 37.738 212.723 40.2229C208.014 42.99 204.971 47.4081 204.374 52.3428C203.185 62.1707 211.483 70.9896 223.265 72.4212C223.521 72.4526 223.776 72.4677 224.028 72.4677C227.154 72.4677 229.86 70.1271 230.245 66.9386C230.662 63.4927 228.212 60.3607 224.774 59.9432C219.875 59.3483 216.537 56.2382 216.825 53.8556C216.992 52.4765 218.302 51.5156 219.068 51.0656C220.914 49.9809 223.439 49.536 225.995 49.8472C227.038 49.9742 228.078 50.0364 229.109 50.0364C233.323 50.0364 237.38 48.9903 240.777 46.9934C245.486 44.2263 248.529 39.8082 249.126 34.8735C249.723 29.9377 247.821 24.9191 243.908 21.1043ZM441.684 130.804C444.58 128.901 445.389 125.005 443.489 122.102C422.471 89.9844 438.585 67.3448 448.212 53.8185C449.383 52.1736 450.489 50.6194 451.391 49.2258C453.276 46.3138 452.449 42.4218 449.542 40.5329C446.636 38.644 442.753 39.4734 440.868 42.3854C440.117 43.5468 439.136 44.9241 438 46.5198C428.028 60.5313 407.277 89.6872 433 128.996C433.571 129.869 434.35 130.586 435.266 131.081C436.183 131.577 437.209 131.836 438.25 131.834C439.471 131.835 440.665 131.477 441.684 130.804Z"
```

Comment:
Isn't there some nicer way how to load svgs?

Reply:
Yes, there is